deid/utils/actions.py

```diff
@@ -26,7 +27,7 @@ def parse_value(dicom, value, item=None, field=None, funcs=None):
         item = dict()
 
     # Does the user want a custom value?
-    if re.search("[:]", value):
+    if re.search("(^var:)|(^func:)|(^deid_func:)", value):
```

Comment:
@vsoch - Modified this to be explicit.  By only searching for the colon, we were incorrectly sending URL/URI values down this path.  Noticed this with the addition of the tests for the UR VR.

Reply:
doh. Thanks for the catch.